src/Simplify.elm

```diff
@@ -4220,6 +4221,30 @@ comparisonWithEmptyChecks isEqual checkInfo =
                     Nothing
 
 
+{-| If the expression is [], String.empty
```

Comment:
String.empty? Did you mean Array.empty and friends?

Reply:
Yes. Yes I did